src/blocks/woo-portal/search/assets/scripts/vue/TheDetail.vue

```diff
@@ -308,26 +310,26 @@ export default {
 .woo-portal-detail__download-circle {
   grid-area: download-icon;
   aspect-ratio: 1;
-  background: var(--owp-color-search-button);
+  background: #000;
```

Comment:
Just checking, this is always these colors instead the variable?

Reply:
exactly